prbench/src/prbench/envs/tidybot/objects.py

```diff
@@ -268,3 +269,299 @@ def __repr__(self) -> str:
             f"Cube(name='{self.name}', joint_name='{self.joint_name}', "
             f"size={self.size}, rgba='{self.rgba}', mass={self.mass})"
         )
+
+
+class MujocoFixture:
```

Comment:
we should make sure to include these in the constant initial object state (see https://github.com/Princeton-Robot-Planning-and-Learning/prpl-mono/blob/6dea817bc6930866f3052e94f1fa08afead6cf17/prbench/src/prbench/core.py#L120)

Reply:
oh yes that makes sense, thanks for being careful about this! I think we do need to include fixtures in the object-centric state to enable generalization, as you said. that's been our pattern in the other environments so far